ichnaea/service/search/schema.py

```diff
@@ -38,9 +38,16 @@ class WifisSchema(SequenceSchema):
     wifi = WifiSchema()
 
 
+class FallbackSchema(MappingSchema):
+
+    lacf = SchemaNode(Integer(), missing=None)
+    ipf = SchemaNode(Integer(), missing=None)
```

Comment:
Would it work to make this a boolean node? Does that convert `0/1` and JSON `true/false` to True/False?